activation/e2e/builds_atx_v2_test.go

```diff
@@ -87,24 +88,13 @@ func TestBuilder_SwitchesToBuildV2(t *testing.T) {
 	}, 10*time.Second, 100*time.Millisecond, "timed out waiting for connection")
 
 	// ensure that genesis aligns with layer timings
-	genesis := time.Now().Add(layerDuration).Round(layerDuration)
-	layerDuration := 2 * time.Second
+	genesis := time.Now().Round(layerDuration)
```

Comment:
The previous rounding ensured that genesis is in the future, this might now set genesis into the past.

Reply:
The genesis doesn't need to be in the future